components/tidb_query/src/expr/ctx.rs

```diff
@@ -53,6 +53,12 @@ bitflags! {
     }
 }
 
+impl Default for Flag {
```

Comment:
The `bitflags!` macro will generate an `empty` method for `Flag`, so we can use `Flag::empty()` directly instead of implementing the `Default` trait for it.